modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -30,6 +30,20 @@
 
 The following premium plugin updates were released alongside {productname} 6.5.
 
+=== Advanced Templates
+
+The {productname} 6.5 release includes an accompanying release of the **Advanced Templates** premium plugin.
+
+**Advanced Templates** 1.2.0 includes one addition:
+
+=== New `advtemplate_templates` configuration option to define an immutable list of predefined templates.
+//#TINY-9755
+
+In {productname} 6.5, the New `advtemplate_templates` configuration option was introduced. This feature allows users to define an unchangeable list of predefined templates. It is designed for situations where the template list should remain static. Instead of setting up a persistent template store and configuring a complex plugin, users can simply specify a static list of templates that can be easily inserted using the Templates dialog. This option simplifies the process for scenarios where a fixed set of templates is required.
```

Comment:
I mean it is not the plugin is complex itself but the plugin configuration requires numerous options to be set
```suggestion
In {productname} 6.5, the New `advtemplate_templates` configuration option was introduced. This feature allows users to define an unchangeable list of predefined templates. It is designed for situations where the template list should remain static. Instead of setting up a persistent template store and dealing with a lengthy plugin configuration, users can simply specify a static list of templates that can be easily inserted using the Templates dialog. This option simplifies the process for scenarios where a fixed set of templates is required.
```